atlas/main_test.go

```diff
@@ -67,3 +69,19 @@ func TestGetVersion(t *testing.T) {
 		t.Errorf("expected response to be status 200, but got %d: %v", resp.StatusCode, resp)
 	}
 }
+
+func TestFormatting(t *testing.T) {
+	cmd := exec.Command("go", "fmt", "./...")
```

Comment:
fun fact: you can use `cmd.Output` to do this with less lines: https://golang.org/pkg/os/exec/#Cmd.Output

Reply:
Thanks for pointing this out. The test looks a bit clunky, so hopefully `cmd.Output` will clean it up.